packages/material-ui/src/Menu/Menu.js

```diff
@@ -23,10 +23,6 @@ const LTR_ORIGIN = {
 export const styles = {
   /* Styles applied to the `Paper` component. */
   paper: {
-    // specZ: The maximum height of a simple menu should be one or more rows less than the view
-    // height. This ensures a tapable area outside of the simple menu with which to dismiss
-    // the menu.
-    maxHeight: 'calc(100% - 96px)',
```

Comment:
How would we solve the menu overflow issue? Here is a reproduction on https://codesandbox.io/s/material-demo-3k97p?file=/package.json with codesandbox-ci. The options at the bottom can't be reached. This impacts a Select with many options. A "light" version of the problem is described in #11349.